modules/blocks/verify.js

```diff
@@ -573,22 +573,41 @@ Verify.prototype.deleteBlockProperties = function(block) {
 	}
 	if (reducedBlock.totalAmount === 0) {
 		delete reducedBlock.totalAmount;
+	} else {
+		reducedBlock.totalAmount = new bignum(reducedBlock.totalAmount);
 	}
 	if (reducedBlock.totalFee === 0) {
 		delete reducedBlock.totalFee;
+	} else {
+		reducedBlock.totalFee = new bignum(reducedBlock.totalFee);
 	}
 	if (reducedBlock.payloadLength === 0) {
 		delete reducedBlock.payloadLength;
 	}
 	if (reducedBlock.reward === 0) {
 		delete reducedBlock.reward;
+	} else {
+		reducedBlock.reward = new bignum(reducedBlock.reward);
 	}
 	if (reducedBlock.transactions && reducedBlock.transactions.length === 0) {
 		delete reducedBlock.transactions;
+	} else {
+		convertToBigNum(reducedBlock.transactions);
 	}
 	return reducedBlock;
 };
 
+const convertToBigNum = transactions =>
```

Comment:
From where those transactions come from? It's verify module - we should already have those as bignums.